app/client/templates/admin/submissions.html

```diff
@@ -35,18 +40,30 @@
               {{projectTitle}}
             {{/if}}
           </td>
-          <td>
+          <td>          
             {{status}}
           </td>
-          {{#if isAdmin}}
           <td>
             {{#if isSubmitted}}
```

Comment:
This is not working for me, the helper in helpers/spacebars.js is failing cuz this.status doesn't exist it should be this.application.status cuz of the way data is declared in the adminRouter.